src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import style from './Tabs.st.css';
+import classNames from 'classnames';
+import { CONTENT_ALIGNMENT, SKIN, CONTENT_WIDTH } from './constants';
+
+export interface TabsProps {
+  items: TabItem[];
```

Comment:
every one of the props should have a comment on top of it
for example - `/** tabs to be displayed */`